meta-lmp-bsp/wic/sdimage-stm32mp157c-dk2-trusted-sota.wks.in

```diff
@@ -0,0 +1,19 @@
+# short-description: Create SD card image with a boot partition
+# long-description: Creates a partitioned SD card image
+#
+#  - ----- --------- -------------- ---------------
+# | | TFA | u-boot  |    /bootfs   |   rootfsfs   |
+#  - ----- --------- -------------- ---------------
+# ^ ^     ^         ^              ^
+# | |     |         |              |
+# 0 17kB 529kB     4MB           68MB
+#
+# Warning: the first stage of boot (here fsbl1, fsbl2, ssbl) MUST be on GPT partition to be detected.
+#
+part fsbl1 --source rawcopy --fstype=ext4 --fsoptions "noauto" --part-name=fsbl1 --sourceparams="file=${DEPLOY_DIR_IMAGE}/arm-trusted-firmware/tf-a-stm32mp157c-dk2-trusted.stm32" --ondisk mmcblk --part-type 0x8301 --fixed-size 256K --align 17
+part fsbl2 --source rawcopy --fstype=ext4 --fsoptions "noauto" --part-name=fsbl2 --sourceparams="file=${DEPLOY_DIR_IMAGE}/arm-trusted-firmware/tf-a-stm32mp157c-dk2-trusted.stm32" --ondisk mmcblk --part-type 0x8301 --fixed-size 256K
+part ssbl  --source rawcopy --fstype=ext4 --fsoptions "noauto" --part-name=ssbl --sourceparams="file=${DEPLOY_DIR_IMAGE}/bootloader/u-boot-stm32mp157c-dk2-trusted.stm32" --ondisk mmcblk --part-type 0x8301 --fixed-size 2048K
```

Comment:
Are they all suppose to be ext4? I thought that boot partition was fat/vfat/

Reply:
In this wks we don't have a vfat partition (for boot), these are special partitions for atf and u-boot (since this is gpt, they don't use an offset).

I didn't try if other fstypes would work the same way as I just copied that logic from https://github.com/STMicroelectronics/meta-st-stm32mp/blob/dunfell/wic/sdcard-stm32mp157c-dk2-trusted-vendorfs-1GB.wks.in.